src/modules/rating/controllers/admin-rating.controller.ts

```diff
@@ -39,4 +43,25 @@ export class AdminRatingController {
 			throw new BadRequestException('Failed to update rating status')
 		}
 	}
+
+	@Get('/status/:status')
+	@ApiResponse({
+		status: 200,
+		type: FilteredByStatusDTO,
+	})
+	async getRatingsByStatus(
+		@Param('status') status: string,
+	): Promise<Rating[]> {
```

Comment:
- Should not return response body as an Array. Should nest the array in a field (for example: **items**)
- There is no pagination